src/resources/js/components/AccountEdit.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <v-container>
+
+  <v-alert
+    v-model="message.show"
+    :class="message.type"
+    dismissible
+    >
+      {{ message.content }}
+    </v-alert>
+
+  <h2>My Account</h2>
+
+  <v-form
+    ref="form"
+    v-model="valid"
+    lazy-validation
+  >
+    <v-text-field
+      v-model="updateUserInfo.fname"
+      :counter="10"
+      :rules="nameRules"
+      label="firstName"
+    ></v-text-field>
+
+    <v-text-field
+      v-model="updateUserInfo.lname"
+      :counter="10"
+      :rules="nameRules"
+      label="lastName"
+    ></v-text-field>
+
+    <v-text-field
+      v-model="updateUserInfo.email"
+      :rules="emailRules"
+      label="E-mail"
+    ></v-text-field>
+
+    <v-text-field
+      v-model="updateUserInfo.password"
+      :append-icon="showPassword ? 'mdi-eye' : 'mdi-eye-off'"
+      :type="showPassword ? 'text' : 'password'"
+      label="password"
+      value=""
+      class="input-group--focused"
+      @click:append="showPassword = !showPassword"
+    ></v-text-field>
+
+    <v-btn
+      :disabled="!valid"
+      color="success"
+      class="mr-4"
+      @click="submitChange"
+    >
+      Submit
```

Comment:
should be in french

Reply:
corrected on [97ccb](https://github.com/kayoumido/HEIGVD-PRO-A-11/pull/123/commits/97ccba253888a5236af0a455f5f24bbd8bed6b21)